test/fixtures/blockcerts-3.0-alpha.json

```diff
@@ -2,9 +2,31 @@
   "@context": [
```

Comment:
Not a huge deal but imo, if possible, I'd like to leave a basic blockcerts v3 in there without the additional contexts just to make sure the base works correctly.